pkg/query/build_tree_test.go

```diff
@@ -572,7 +580,7 @@ func TestBuildTreeSubrelationHandling(t *testing.T) {
 
 	t.Run("Base Relation with Ellipsis Subrelation", func(t *testing.T) {
```

Comment:
This test `"Base Relation with Ellipsis Subrelation"` seems to be exactly the same as `"Base Relation Without Subrelations Disabled"`. I mean, the former has `groupDef` in the schema, but it's not related to `docDef` at all and you don't build an iterator off it, so it's like it doesn't exist.